scripts/skills/actives/rf_cheap_trick_skill.nut

```diff
@@ -0,0 +1,96 @@
+this.rf_cheap_trick_skill <- ::inherit("scripts/skills/skill", {
+	m = {
+		MeleeSkillModifier = 25,
+		RangedSkillModifier = 25,
+		DamageMult = 0.50,
+
+		// Private
+		WasUsedThisTurn = false,
+	},
+	function create()
+	{
+		this.m.ID = "actives.rf_cheap_trick";
+		this.m.Name = "Cheap Trick";
+		this.m.Description = "Deceive your enemies and strike with precision, sacrificing power for accuracy with your next attack this turn.";
+		// Todo: create icons
+		// this.m.Icon = "skills/rf_cheap_trick_skill.png";
+		// this.m.IconDisabled = "skills/rf_cheap_trick_skill_bw.png";
+		// this.m.Overlay = "rf_cheap_trick_skill";
+		this.m.SoundOnUse = [];
+		this.m.Type = ::Const.SkillType.Active;
+		this.m.Order = ::Const.SkillOrder.Any;
+		this.m.IsSerialized = false;
+		this.m.IsActive = true;
+		this.m.ActionPointCost = 0;
+		this.m.FatigueCost = 10;
+	}
+
+	function getTooltip()
+	{
+		local ret = this.skill.getDefaultUtilityTooltip();
+
+		if (this.m.MeleeSkillModifier != 0)
+		{
+			ret.push({
+				id = 10,
+				type = "text",
+				icon = "ui/icons/melee_skill.png",
+				text = ::Reforged.Mod.Tooltips.parseString("Gain " + ::MSU.Text.colorizeValue(this.m.MeleeSkillModifier) + " [Melee Skill|Concept.MeleeSkill]"),
+			});
+		}
+
+		if (this.m.RangedSkillModifier != 0)
+		{
+			ret.push({
+				id = 11,
+				type = "text",
+				icon = "ui/icons/ranged_skill.png",
+				text = ::Reforged.Mod.Tooltips.parseString("Gain " + ::MSU.Text.colorizeValue(this.m.RangedSkillModifier) + " [Ranged Skill|Concept.RangedSkill]"),
+			});
+		}
+
+		if (this.m.DamageMult != 1.0)
+		{
+			ret.push({
+				id = 12,
+				type = "text",
+				icon = "ui/icons/damage_dealt.png",
+				text = ::Reforged.Mod.Tooltips.parseString("Deal " + ::MSU.Text.colorizeMult(this.m.DamageMult) + " less damage"),
+			});
+		}
```

Comment:
I made a rework that still mentions Melee Skill and Ranged Skill separately.

I rather not use this vague term "chance to hit" unless we introduce a new concept for it that explicitly states that this means MeleeSkill AND RangedSkill and **will** scale with multipliers to those.

Reply:
I agree that that the `Chance to hit` is badly used in vanilla tooltips because it's also affected by multipliers. But saying that you have +x Melee Skill for the next attack isn't fully clear either. I'm afraid many players will interpret it as you gain +25 Melee Skill and +25 Ranged Skill **until you do another attack**. They will use this skill, and check their bro's stats and open a bug report that see I am not getting the stats.

Perhaps it could be worded like `during` instead of `for` the next attack. I'm not sure if that will solve it fully either.

The hitchance tooltip keeps the behavior clear for the user's side and while I agree that mechanically it is not 100% exact, I don't think that the alternative is a good solution either. We could brainstorm this to come up with a better way of representing this in tooltips. 

Idk, I don't have a perfect solution for this at this moment.

For this build I'd say change to hit chance for now (to be consistent with vanilla skills that do the same). Then in the future when we have an actual solution to this tooltip issue we could go through and update these including the vanilla ones.

Reply:
what icon do you recommend for a tooltip line that grants "chance to hit"?

Reply:
The same one that vanilla uses: `"ui/icons/hitchance.png`